trio/_tests/test_windows_pipes.py

```diff
@@ -14,24 +16,51 @@
 else:
     pytestmark = pytest.mark.skip(reason="windows only")
     pipe: Any = None
-    PipeSendStream: Any = None
-    PipeReceiveStream: Any = None
+    _handle: Any = None
+    kernel32: Any = None
+    from .._abc import ReceiveStream, SendStream
+
+    class _fake_recieve(ReceiveStream):
```

Comment:
Hmm typo here. I'm on the edge on whether this making a class really is worth it over just assigning a `None` and type ignoring it... (or... does it even need an assignment?)

Reply:
I always forget how to spell receive, I can fix that.

Reply:
Fixed in bc72284c

Reply:
The only reason we have a subclass here is because the default `ReceiveStream` and `SendStream`'s init takes no arguments, while the `Pipe` versions require an integer.

Reply:
wait... because `pytestmark` https://docs.pytest.org/en/7.1.x/example/markers.html#marking-whole-classes-or-modules is used, the tests aren't run on non-windows at all (that surely deserves a comment anywhere it's used IMHO), so can just skip typechecking the file on non-windows.

Reply:
Made to skip typechecking on non-windows in 6fc51bd

Reply:
awesome :+1: